src/backend/joanie/core/templates/debug/sentry_decrypt.html

```diff
@@ -0,0 +1,12 @@
+<a href={% url "admin:index" %}>Back to admin</a>
+
+<form action="{% url "sentry-decrypt" %}" method="post">
+    {% csrf_token %}
+    <textarea name="encrypted" rows="20" cols="100"></textarea>
+    <input type="submit" value="Decrypt">
+</form>
+
+{% if decrypted %}
+    <h2>Decrypted</h2>
+    <pre>{{ decrypted | pprint }}</pre>
+{% endif %}
```

Comment:
We could extend from `admin/base.html` template.

```suggestion
{% extends "admin/base.html" %}

{% block content %}
    <form action="{% url "sentry-decrypt" %}" method="post">
        {% csrf_token %}
        <textarea name="encrypted" rows="20" cols="100"></textarea>
        <input type="submit" value="Decrypt">
    </form>

    {% if decrypted %}
        <h2>Decrypted</h2>
        <pre>{{ decrypted | pprint }}</pre>
    {% endif %}
{% endblock %}

```

<img width="800" alt="image" src="https://github.com/openfun/joanie/assets/9265241/e5f7ae81-065e-4d6c-baae-2f31c473832c">